src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -419,6 +429,24 @@ def prepare_logs_argument(c):
                    help="Disable Application Insights.",
                    validator=validate_app_insights_parameters)
 
+    with self.argument_context('spring-cloud build-service builder') as c:
+        c.argument('service', service_name_type, validator=only_support_enterprise)
+
+    for scope in ['create', 'update']:
+        with self.argument_context('spring-cloud build-service builder {}'.format(scope)) as c:
+            c.argument('builder_json', type=str, help="The JSON array of builder.", validator=validate_builder_resource)
+            c.argument('builder_file', type=str, help="The file path of JSON array of builder.", validator=validate_builder_resource)
+
+    with self.argument_context('spring-cloud build-service builder create') as c:
+        c.argument('name', type=str, help="The builder name.", validator=validate_builder_create)
+
+    with self.argument_context('spring-cloud build-service builder update') as c:
+        c.argument('name', type=str, help="The builder name.", validator=validate_builder_update)
+
+    for scope in ['show', 'delete']:
+        with self.argument_context('spring-cloud build-service builder {}'.format(scope)) as c:
+            c.argument('name', type=str, help="The builder name.")
```

Comment:
`type=str` is the default setting, so we don't need an explicit declaration

Reply:
OK, deleted the extra setting